dt/network/module.json

```diff
@@ -122,6 +122,14 @@
                 ["All", "all"],
                 ["Trailing", "trailing"]
             ]
+        },
+        {
+            "type": "setting",
+            "title": "Auto-reload generated CSS",
```

Comment:
This setting was missing (because we are not loading sources panel) causing saving to file fail.